client/jetpack-cloud/sections/agency-dashboard/sites-overview/index.tsx

```diff
@@ -191,6 +191,36 @@ export default function SitesOverview() {
 		} );
 	}, [ selectedLicensesSiteId, selectedLicenses ] );
 
+	const renderAddSiteIssueLicenseButtons = () => {
+		return (
+			<div className="sites-overview__add-site-issue-license-buttons">
+				<Button
+					className="sites-overview__issue-license-button"
+					href="/partner-portal/issue-license"
+					onClick={ () =>
+						dispatch(
+							recordTracksEvent( 'calypso_jetpack_agency_dashboard_issue_license_button_click', {} )
```

Comment:
I don't think an empty object is needed here

```suggestion
							recordTracksEvent( 'calypso_jetpack_agency_dashboard_issue_license_button_click' )
```